### PR TITLE
add minimum population requirement for kill objectives to changeling and wizzard

### LINKED ISSUE #1884: No-kill traitor option does not affect other antagonists.
PR#1617 changed how syndicate traitors do not have a kill objective if the current player count is under a value. However this does not seem to affect other antagonists including changelings, heretics, space ninja and wizard. This makes balance very advantageous to antags sided because they are made for a crew that has a full security staffing such as on space stations. But on ships it is often more lacking because the need of a munitions crew instead.

### PR DESCRIPTION
**Pull request by Kenionatus**
## About The Pull Request

Only give kill objectives to wizards and changelings if the population is above MIN_POP_KILL_OBJECTIVES from config\game_options.txt
Fixes #1884
 

## Why It's Good For The Game

Brings wizards and changelings in line with traitors. Consistency is good because people get upset about inconsistencies. Also, lowpop murder is lame.

## Changelog
:cl:
tweak: No longer give kill objectives to wizards and changelings on low pop.
/:cl:

### CODE REVIEW

**Comment by Kenionatus** [comment]
Wasn't able to really test it because I don't know of any way to create dummy players. It does however not crash if I local host, join as only player and make myself wizard or changeling. Also didn't give me any kill objectives, tho there isn't anyone to kill so that might not have any connection to my code.

**Comment by Kenionatus** [comment]
Fixes #1884

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]
Mark your changes with //nsv13 if they're not in the nsv folder

File: code/modules/antagonists/changeling/changeling.dm [hunk 1]
```diff
@@ -416,36 +416,38 @@
...
+				log_objective(owner, maroon_objective.explanation_text)
+
+				if (!(locate(/datum/objective/escape) in objectives) && escape_objective_possible)
+					var/datum/objective/escape/escape_with_identity/identity_theft = new
+					identity_theft.owner = owner
+					identity_theft.target = maroon_objective.target
+					identity_theft.update_explanation_text()
+					objectives += identity_theft
+					log_objective(owner, identity_theft.explanation_text)
+					escape_objective_possible = FALSE
```
[the remark this answers is not in the document]
Comment (Kenionatus) [reply code/modules/antagonists/changeling/changeling.dm:446]: I feel like that might be slightly misleading in the changeling case. Everything except the commented lines is just a shift in indentation that stems from those and the diff shows a much bigger change than logically happened imo. If you think it's still worth strictly sticking to an end comment I'll not object tho.

Wizzard file is a clear case for an "//nsv13 end". Added in [this commit](https://github.com/BeeStation/NSV13/commit/bf14d0e271af637d249a2b058141adf1c46844b2).

**Review by Bokkiewokkie — APPROVED** [review]

**Pull request merged**